### PR TITLE
[shopsys] Release of version 12.0

### PR DESCRIPTION
**Pull request by TomasLudvik**
| Q             | A
| ------------- | ---
|Description, the reason for the PR| New major version with TWIG storefront that is marked as LTS.
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,32 @@ Packages are formatted by release version. You can see all the changes done to p
...
+
+### :cloud: Infrastructure
+* [shopsys] improvements for deployment/running in cluster by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2609
+* [shopsys] CDN support by @grossmannmartin https://github.com/shopsys/shopsys/pull/2602
+
+### :placard: Other Changes
+* [coding-standards] updated slevomat coding standards by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2646
+* [shopsys] inheritdoc docblock is now unified and fixed automatically by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2612
+* [s3-bridge] fixed bundle conventions by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2645
+* [framework] fixed seoRobotsTxtContent null value in settings by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2731
```
Comment (grossmannmartin) [inline CHANGELOG.md]: this is already mentioned in bug fixes section

File: CHANGELOG.md [hunk 2]
```diff
@@ -27,6 +27,32 @@ Packages are formatted by release version. You can see all the changes done to p
...
+* [framework] encapsulation of AdditionalImageData by @pk16011990 in https://github.com/shopsys/shopsys/pull/1934
+* [shopsys] updated codeception to version 5 by @TomasLudvik https://github.com/shopsys/shopsys/pull/2611
+
+### :cloud: Infrastructure
+* [shopsys] improvements for deployment/running in cluster by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2609
+* [shopsys] CDN support by @grossmannmartin https://github.com/shopsys/shopsys/pull/2602
+
+### :placard: Other Changes
+* [coding-standards] updated slevomat coding standards by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2646
+* [shopsys] inheritdoc docblock is now unified and fixed automatically by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2612
```
Comment (grossmannmartin) [inline CHANGELOG.md]: I would place these two into the "Developer experience and refactoring" section

File: CHANGELOG.md [hunk 3]
```diff
@@ -27,6 +27,32 @@ Packages are formatted by release version. You can see all the changes done to p
...
+* [s3-bridge] fixed bundle conventions by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2645
```
Comment (grossmannmartin) [inline CHANGELOG.md]: I think it's not necessary to mention this one, or?

File: upgrade/UPGRADE-v12.1.0-dev.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# [Upgrade from v12.0.0 to v12.1.0-dev](https://github.com/shopsys/shopsys/compare/v12.0.0...12.0)
```
Comment (grossmannmartin) [inline upgrade/UPGRADE-v12.1.0-dev.md]: Shouldn't be the branch named 12.1?

**Review by grossmannmartin — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (TomasLudvik) [reply CHANGELOG.md]: Removed

File: CHANGELOG.md (hunk 2 above)
Comment (TomasLudvik) [reply CHANGELOG.md]: Moved

File: CHANGELOG.md (hunk 3 above)
Comment (TomasLudvik) [reply CHANGELOG.md]: Removed

File: upgrade/UPGRADE-v12.1.0-dev.md (hunk 4 above)
Comment (TomasLudvik) [reply upgrade/UPGRADE-v12.1.0-dev.md]: Hmm, interesting idea, I have created branch and changed this.

**Review by grossmannmartin — APPROVED** [review]

**Review by pk16011990 — APPROVED** [review]

**Pull request closed without merging**